mailman_server.rb: process with pid 6565 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-30T21:53:07.210231 #6565]  INFO -- : Mailman v0.7.0 started
I, [2016-10-30T21:53:07.210431 #6565]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-30T21:53:07.226392 #6565]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-10-30T22:20:54.827969 #6565]  INFO -- : Got new message from 'mailer-daemon@googlemail.com' with subject 'Delivery Status Notification (Failure)'.
mailman_server.rb: process with pid 2104 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T09:54:53.159182 #2104]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T09:54:53.159690 #2104]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T09:54:53.176616 #2104]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 2817 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T10:13:15.137330 #2817]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T10:13:15.137542 #2817]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T10:13:15.154822 #2817]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 3468 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T10:26:36.938434 #3468]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T10:26:36.938581 #3468]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T10:26:36.957636 #3468]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 3596 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T10:29:09.500635 #3596]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T10:29:09.500955 #3596]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T10:29:09.519444 #3596]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 4291 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T10:43:21.406668 #4291]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T10:43:21.406819 #4291]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T10:43:21.485210 #4291]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 4394 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T10:44:23.492132 #4394]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T10:44:23.492396 #4394]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T10:44:23.515467 #4394]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 4965 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T10:45:48.287386 #4965]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T10:45:48.287513 #4965]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T10:45:48.303407 #4965]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 5222 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T11:10:54.727697 #5222]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T11:10:54.727942 #5222]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T11:10:54.739929 #5222]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 6898 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T11:16:04.527221 #6898]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T11:16:04.527414 #6898]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T11:16:04.548668 #6898]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 6992 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T11:16:17.796346 #6992]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T11:16:17.796531 #6992]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T11:16:17.822683 #6992]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 7086 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T11:17:21.908580 #7086]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T11:17:21.908712 #7086]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T11:17:21.925233 #7086]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 7310 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T11:36:03.679127 #7310]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T11:36:03.679324 #7310]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T11:36:03.712227 #7310]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 7404 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T11:37:16.243977 #7404]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T11:37:16.244149 #7404]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T11:37:16.291492 #7404]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 7500 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-10-31T11:38:50.155411 #7500]  INFO -- : Mailman v0.7.0 started
I, [2016-10-31T11:38:50.278246 #7500]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-10-31T11:38:50.304608 #7500]  INFO -- : Polling enabled. Checking every 20 seconds.
/usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/pop.rb:1006:in `check_response_auth': -ERR [SYS/TEMP] Temporary system problem. Please try again later. (Net::POPAuthenticationError)
	from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/pop.rb:905:in `auth'
	from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/pop.rb:567:in `do_start'
	from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/pop.rb:537:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/receiver/pop3.rb:31:in `connect'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:132:in `block in polling_loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:130:in `loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:130:in `polling_loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:83:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:11:in `run'
	from /home/ubuntu/project/remindme1/scripts/mailman_server.rb:19:in `<top (required)>'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:219:in `load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:219:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:158:in `block (2 levels) in start_all'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:157:in `fork'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:157:in `block in start_all'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:156:in `each'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:156:in `start_all'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:66:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
mailman_server.rb: process with pid 1653 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-01T20:09:38.974524 #1653]  INFO -- : Mailman v0.7.0 started
I, [2016-11-01T20:09:38.974961 #1653]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-01T20:09:38.992356 #1653]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 1622 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-02T20:27:35.096022 #1622]  INFO -- : Mailman v0.7.0 started
I, [2016-11-02T20:27:35.096394 #1622]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-02T20:27:35.112061 #1622]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 2095 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-03T20:33:09.896908 #2095]  INFO -- : Mailman v0.7.0 started
I, [2016-11-03T20:33:09.897302 #2095]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-03T20:33:10.080458 #2095]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 2080 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-04T19:22:34.481418 #2080]  INFO -- : Mailman v0.7.0 started
I, [2016-11-04T19:22:34.481791 #2080]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-04T19:22:34.499371 #2080]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 2229 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-04T19:29:52.669271 #2229]  INFO -- : Mailman v0.7.0 started
I, [2016-11-04T19:29:52.669436 #2229]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-04T19:29:52.681351 #2229]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 2333 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-04T19:30:36.465348 #2333]  INFO -- : Mailman v0.7.0 started
I, [2016-11-04T19:30:36.465523 #2333]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-04T19:30:36.476797 #2333]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 3029 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-04T20:42:27.648238 #3029]  INFO -- : Mailman v0.7.0 started
I, [2016-11-04T20:42:27.648422 #3029]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-04T20:42:27.660658 #3029]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 3219 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-04T20:49:20.049534 #3219]  INFO -- : Mailman v0.7.0 started
I, [2016-11-04T20:49:20.061278 #3219]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-04T20:49:20.076007 #3219]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 2606 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-05T00:11:48.548959 #2606]  INFO -- : Mailman v0.7.0 started
I, [2016-11-05T00:11:48.549486 #2606]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-05T00:11:48.572148 #2606]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 3178 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-05T01:05:47.387790 #3178]  INFO -- : Mailman v0.7.0 started
I, [2016-11-05T01:05:47.388006 #3178]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-05T01:05:47.396912 #3178]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 3765 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-05T02:06:01.182028 #3765]  INFO -- : Mailman v0.7.0 started
I, [2016-11-05T02:06:01.182230 #3765]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-05T02:06:01.192424 #3765]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 6340 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-05T03:04:29.209762 #6340]  INFO -- : Mailman v0.7.0 started
I, [2016-11-05T03:04:29.209981 #6340]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-05T03:04:29.278740 #6340]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 1026 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-06T15:44:40.999284 #1026]  INFO -- : Mailman v0.7.0 started
I, [2016-11-06T15:44:40.999861 #1026]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-06T15:44:41.019888 #1026]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 4228 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-06T18:57:34.653895 #4228]  INFO -- : Mailman v0.7.0 started
I, [2016-11-06T18:57:34.654064 #4228]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-06T18:57:34.671561 #4228]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 4420 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-06T19:11:52.795320 #4420]  INFO -- : Mailman v0.7.0 started
I, [2016-11-06T19:11:52.795534 #4420]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-06T19:11:52.809784 #4420]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 2085 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T20:21:40.653849 #2085]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T20:21:40.654200 #2085]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T20:21:40.670462 #2085]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-11-07T20:39:07.246035 #2085]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '10.40pm 07 Nov 2017#Test 10.40pm 07 Nov 2017'.
I, [2016-11-07T20:39:29.522422 #2085]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '10.40pm 07 November 2017#10.40pm 07 November 2017'.
I, [2016-11-07T20:40:12.346281 #2085]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2017 22.40#07/11/2017 22.40 Test'.
I, [2016-11-07T20:41:36.879128 #2085]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 10.40pm#Tester test'.
I, [2016-11-07T20:42:41.582746 #2085]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 10.44pm#Tester test'.
I, [2016-11-07T20:44:06.858202 #2085]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 22.44#test'.
I, [2016-11-07T20:44:50.035593 #2085]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 22.46#test'.
I, [2016-11-07T20:47:21.710478 #2085]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 22.49#test'.
I, [2016-11-07T20:48:46.588753 #2085]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 22.50#test'.
mailman_server.rb: process with pid 2464 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T20:49:46.282924 #2464]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T20:49:46.283092 #2464]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T20:49:46.299348 #2464]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-11-07T20:50:08.220573 #2464]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 22.51#test'.
E, [2016-11-07T20:50:08.993209 #2464] ERROR -- : No such file or directory @ rb_sysopen - log/exceptions.log
/usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/pop.rb:528:in `start': POP session already started (IOError)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/receiver/pop3.rb:31:in `connect'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:132:in `block in polling_loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:130:in `loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:130:in `polling_loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:83:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:11:in `run'
	from /home/ubuntu/project/remindme1/scripts/mailman_server.rb:19:in `<top (required)>'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:219:in `load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:219:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:158:in `block (2 levels) in start_all'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:157:in `fork'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:157:in `block in start_all'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:156:in `each'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:156:in `start_all'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:66:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
mailman_server.rb: process with pid 2579 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T20:51:04.831704 #2579]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T20:51:04.832036 #2579]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T20:51:04.845534 #2579]  INFO -- : Polling enabled. Checking every 20 seconds.
mailman_server.rb: process with pid 2686 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T20:52:27.254944 #2686]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T20:52:27.255175 #2686]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T20:52:27.267369 #2686]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-11-07T20:52:50.105805 #2686]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 22.55#test'.
/usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/validations.rb:78:in `raise_validation_error': Validation failed: Schedule must be higher or equal to today (ActiveRecord::RecordInvalid)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/validations.rb:50:in `save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/attribute_methods/dirty.rb:30:in `save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:324:in `block in save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:395:in `block in with_transaction_returning_status'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/connection_adapters/abstract/database_statements.rb:232:in `block in transaction'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/connection_adapters/abstract/transaction.rb:189:in `within_new_transaction'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/connection_adapters/abstract/database_statements.rb:232:in `transaction'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:211:in `transaction'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:392:in `with_transaction_returning_status'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:324:in `save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/suppressor.rb:45:in `save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/persistence.rb:51:in `create!'
	from /home/ubuntu/project/remindme1/app/models/remind.rb:113:in `logreminder'
	from /home/ubuntu/project/remindme1/app/models/remind.rb:65:in `get_email'
	from /home/ubuntu/project/remindme1/scripts/mailman_server.rb:21:in `block (2 levels) in <top (required)>'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/router.rb:66:in `instance_exec'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/router.rb:66:in `route'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/message_processor.rb:23:in `block in process'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/middleware.rb:33:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/middleware.rb:38:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/message_processor.rb:22:in `process'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/receiver/pop3.rb:43:in `block in get_messages'
	from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/pop.rb:666:in `each'
	from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/pop.rb:666:in `each_mail'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/receiver/pop3.rb:42:in `get_messages'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:133:in `block in polling_loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:130:in `loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:130:in `polling_loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:83:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:11:in `run'
	from /home/ubuntu/project/remindme1/scripts/mailman_server.rb:19:in `<top (required)>'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:219:in `load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:219:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:158:in `block (2 levels) in start_all'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:157:in `fork'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:157:in `block in start_all'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:156:in `each'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application_group.rb:156:in `start_all'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:66:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
mailman_server.rb: process with pid 2800 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T20:54:11.809104 #2800]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T20:54:11.809407 #2800]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T20:54:11.822101 #2800]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-11-07T20:54:12.982743 #2800]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 22.55#testwettr'.
/usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/validations.rb:78:in `raise_validation_error': Validation failed: Schedule must be higher or equal to today (ActiveRecord::RecordInvalid)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/validations.rb:50:in `save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/attribute_methods/dirty.rb:30:in `save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:324:in `block in save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:395:in `block in with_transaction_returning_status'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/connection_adapters/abstract/database_statements.rb:232:in `block in transaction'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/connection_adapters/abstract/transaction.rb:189:in `within_new_transaction'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/connection_adapters/abstract/database_statements.rb:232:in `transaction'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:211:in `transaction'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:392:in `with_transaction_returning_status'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/transactions.rb:324:in `save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/suppressor.rb:45:in `save!'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/activerecord-5.0.0.1/lib/active_record/persistence.rb:51:in `create!'
	from /home/ubuntu/project/remindme1/app/models/remind.rb:113:in `logreminder'
	from /home/ubuntu/project/remindme1/app/models/remind.rb:65:in `get_email'
	from /home/ubuntu/project/remindme1/scripts/mailman_server.rb:21:in `block (2 levels) in <top (required)>'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/router.rb:66:in `instance_exec'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/router.rb:66:in `route'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/message_processor.rb:23:in `block in process'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/middleware.rb:33:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/middleware.rb:38:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/message_processor.rb:22:in `process'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/receiver/pop3.rb:43:in `block in get_messages'
	from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/pop.rb:666:in `each'
	from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/pop.rb:666:in `each_mail'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/receiver/pop3.rb:42:in `get_messages'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:133:in `block in polling_loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:130:in `loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:130:in `polling_loop'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:83:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/mailman-0.7.0/lib/mailman/application.rb:11:in `run'
	from /home/ubuntu/project/remindme1/scripts/mailman_server.rb:19:in `<top (required)>'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:219:in `load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:219:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
mailman_server.rb: process with pid 2904 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T20:56:09.522883 #2904]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T20:56:09.523146 #2904]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T20:56:09.592726 #2904]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-11-07T20:56:11.344139 #2904]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 22.55#testwettr'.
I, [2016-11-07T20:56:12.305269 #2904]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '11/07/2016 22.55#test'.
I, [2016-11-07T20:56:13.143956 #2904]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '11/07/2016 22.57#test'.
mailman_server.rb: process with pid 1957 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T20:58:36.173728 #1957]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T20:58:36.174002 #1957]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T20:58:36.185606 #1957]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-11-07T20:58:59.047117 #1957]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 23.00#test'.
I, [2016-11-07T21:00:04.867570 #1957]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 23.00#test'.
mailman_server.rb: process with pid 2068 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T21:01:56.130976 #2068]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T21:01:56.131144 #2068]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T21:01:56.147280 #2068]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-11-07T21:01:57.329619 #2068]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 23.00#test'.
mailman_server.rb: process with pid 2182 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T21:04:38.934145 #2182]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T21:04:38.934431 #2182]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T21:04:38.945126 #2182]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-11-07T21:05:01.449148 #2182]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject 'Today 21:10#Test'.
mailman_server.rb: process with pid 2322 started.
mailman_server.rb: warning: no instances running. Starting...
/usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `reopen': Permission denied @ rb_io_reopen - /scripts/mailman_server.rb.output (Errno::EACCES)
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:157:in `redirect_io'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/daemonize.rb:112:in `daemonize'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:163:in `start_load'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:298:in `start'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/controller.rb:69:in `run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:148:in `block in run'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
	from /usr/local/rvm/gems/ruby-2.3.0/gems/daemons-1.2.4/lib/daemons.rb:147:in `run'
	from /home/ubuntu/project/remindme1/scripts/daemons/mailman_ctl.svc:13:in `<main>'
scheduler.rb: warning: no instances running. Starting...
I, [2016-11-07T21:08:33.114191 #2322]  INFO -- : Mailman v0.7.0 started
I, [2016-11-07T21:08:33.114428 #2322]  INFO -- : POP3 receiver enabled (remindmailer@gmail.com@pop.gmail.com).
I, [2016-11-07T21:08:33.126857 #2322]  INFO -- : Polling enabled. Checking every 20 seconds.
I, [2016-11-07T21:08:54.666399 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '07/11/2016 21.30#test'.
I, [2016-11-07T21:10:19.869726 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '9.15pm 7 Nov 2016#9.15pm 7 Nov 2016'.
I, [2016-11-07T21:10:42.184473 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '9.15pm 7 November 2016#9.15pm 7 November 2016'.
I, [2016-11-07T21:11:04.583850 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '9.15pm 7 November 16#9.15pm 7 November 16'.
I, [2016-11-07T21:12:50.506365 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '2016 8 November 2.30am#2016 8 November 2.30am'.
I, [2016-11-07T21:13:33.275368 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '2016-11-08 13:35#2016-11-08 13:35'.
I, [2016-11-07T21:14:15.942219 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '2016/11/8 15:30#2016/11/8 15:30'.
I, [2016-11-07T21:16:22.287184 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '08.11.2016 11:30#08.11.2016 11:30'.
I, [2016-11-07T22:05:28.558501 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '06/06/06/06/06#test'.
I, [2016-11-07T22:06:31.947856 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '13 Janvar 17 17:30#fsgjsd'.
I, [2016-11-07T22:09:19.904867 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '13 Janvar 17 17:30#fsgjsd'.
I, [2016-11-07T22:11:26.394491 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '06/11/2016 5:10#fdgsfdgsdf'.
I, [2016-11-07T22:13:33.117512 #2322]  INFO -- : Got new message from 'a.kuriackovskij@gmail.com' with subject '04/10/2016 14:30#349ngsfdngsdf'.
